sdk/labservices/Microsoft.Azure.Management.LabServices/src/Microsoft.Azure.Management.LabServices.csproj

```diff
@@ -4,12 +4,12 @@
 	<!-- Please do not move/edit code below this line -->
 
 	<PropertyGroup>
-		<Version>4.0.0</Version>
+		<Version>4.1.0</Version>
```

Comment:
I don't see a 4.0.0 released here https://www.nuget.org/packages/Microsoft.Azure.Management.LabServices/, maybe we should leave this as 4.0.0 and release it?  It would seem weird to release starting from 4.1.0.

Reply:
changed to 4.0.0